upt_macports/templates/ruby.Portfile

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.Portfile' %}
+
+{% block portgroup %}
+PortGroup           ruby 1.0
+{% endblock %}
+{% block nameversion %}
```

Comment:
similarly to the Perl template, here you should add: ```ruby.branches       ???```, where ??? should of course be the version or versions. At least for now, in the testing phase, I would only add the latest two version so ```2.5 2.6```. You should probably take a look at [the Ruby PortGroup](_resources/port1.0/group/ruby-1.0.tcl) to see if there is anything else that should be defined in the template.

Reply:
Yeah I saw the Ruby port group, but seeing the existing Ruby Ports I was not sure about `ruby.branches` so did not add them as only one port is currently using that.
Also as far as I can see we cannot automate the detection of supported version as that information is not available in the json unlike python(even if this is unreliable), but we can sure hardcode it for now.

But as almost none of the ports use it, the question is, is it really needed?

Reply:
Yes, I think so. Just as for Python and Perl we want to be able to support different versions, right now it likely defaults to ```ruby.default_branch```. Even if we only add ```2.6``` for new ports (similarly as we only add ```37``` for ```python.versions``` in the Python template), it's still useful to add in the Portfile so that if a new version comes out we can easily add it.